### ISSUE TITLE
Error when running with workspace = CWD

### ISSUE THREAD

**Issue by Lamarcke** [nvim-telescope/telescope-frecency.nvim#117]
I'm getting an error when running this extension with the `workspace` set to `CWD`:
```
Error executing Lua callback: ...e/pack/lazy/opt/plenary.nvim/lua/plenary/async/async.lua:18: The coroutine failed with this message: ...pt/telescope-frecency.nvim/lua/frecency/async_finder.lu
a:35: ...ack/lazy/opt/telescope-frecency.nvim/lua/frecency/fs.lua:54: attempt to call field 'joinpath' (a nil value)                                                                            
stack traceback:                                                                                                                                                                                
        [C]: in function 'error'                                                                                                                                                                
        ...e/pack/lazy/opt/plenary.nvim/lua/plenary/async/async.lua:18: in function 'callback_or_next'                                                                                          
        ...e/pack/lazy/opt/plenary.nvim/lua/plenary/async/async.lua:45: in function 'step'                                                                                                      
        ...e/pack/lazy/opt/plenary.nvim/lua/plenary/async/async.lua:48: in function 'execute'                                                                                                   
        ...e/pack/lazy/opt/plenary.nvim/lua/plenary/async/async.lua:108: in function 'run'                                                                                                      
        ...pt/telescope-frecency.nvim/lua/frecency/async_finder.lua:32: in function 'start'                                                                                                     
        ...lazy/opt/telescope-frecency.nvim/lua/frecency/picker.lua:87: in function 'start'                                                                                                     
        ...zy/opt/telescope-frecency.nvim/lua/frecency/frecency.lua:104: in function 'start'                                                                                                    
        ...k/lazy/opt/telescope-frecency.nvim/lua/frecency/init.lua:13: in function <...k/lazy/opt/telescope-frecency.nvim/lua/frecency/init.lua:11>                                            
        ...e/pack/lazy/opt/telescope.nvim/lua/telescope/command.lua:193: in function 'run_command'                                                                                              
        ...e/pack/lazy/opt/telescope.nvim/lua/telescope/command.lua:253: in function 'load_command'                                                                                             
        ...m/site/pack/lazy/opt/telescope.nvim/plugin/telescope.lua:108: in function <...m/site/pack/lazy/opt/telescope.nvim/plugin/telescope.lua:107>   
```

This started happening after updating, so it may be related to latest commits? Please do tell me if you need more info.

**Comment by delphinus** [comment]
Sorry, I did not know the func `vim.fs.joinpath` does not exist in v0.9.x. I did a fix in #117. Check it again.

**Comment by Lamarcke** [comment]
> Sorry, I did not know the func `vim.fs.joinpath` does not exist in v0.9.x. I did a fix in #117. Check it again.

Working as expected, thank you very much!
